addons/web/controllers/main.py

```diff
@@ -1049,7 +1049,7 @@ def _content_image(self, xmlid=None, model='ir.attachment', id=None, field='data
             if not (width or height):
                 width, height = odoo.tools.image_guess_size_from_field_name(field)
 
-        image_base64 = image_process(image_base64, (width, height), crop=crop)
+        image_base64 = image_process(base64_source=image_base64, size=(int(width), int(height)), crop=crop)
```

Comment:
base64_source is not yet a named param in master

Reply:
@JKE-be Done, also changed the others, including enterprise https://github.com/odoo/enterprise/pull/4389